File: GithubGists/app/src/main/java/ru/avystavkin/githubgists/screen/base/holders/BaseViewHolder.java
package ru.avystavkin.githubgists.screen.base.holders;

import android.view.View;

import androidx.recyclerview.widget.RecyclerView;
import butterknife.ButterKnife;

public abstract class BaseViewHolder extends RecyclerView.ViewHolder {
    public BaseViewHolder(View itemView) {
        super(itemView);
        ButterKnife.bind(this, itemView);
    }
}

File: GithubGists/app/src/main/java/ru/avystavkin/githubgists/screen/base/holders/BaseViewHolder.kt
package ru.avystavkin.githubgists.screen.base.holders

import android.view.View

import androidx.recyclerview.widget.RecyclerView
import butterknife.ButterKnife

abstract class BaseViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    init {
        ButterKnife.bind(this, itemView)
    }
}
